py/janitor/bzr_store.py

```diff
@@ -21,6 +21,7 @@
 import logging
 import os
 import sys
+import uvloop
```

Comment:
please don't move these around - uvloop is intentionally only included from main() so that e.g. tests don't need it. Nothing else should rely on this.

Reply:
Gotcha. What about `./py/janitor/runner.py` then?
Should that be moved to main()? https://github.com/jelmer/janitor/blob/main/py/janitor/runner.py#L43